scripts/change-log-generator.js

```diff
@@ -88,7 +67,10 @@ function getReleaseBranch() {
  */
 function getReleaseBranches() {
   return shell
-    .exec(GIT_BRANCH, { silent: true })
+    .exec(
+      `git branch --remotes --list --sort='-creatordate' '${constants.RELEASE_BRANCH_PREFIX}*'`,
```

Comment:
I decided to have the commands in place instead of as a constant - I feel this is easier to read.